pdfminer/layout.py

```diff
@@ -116,9 +116,12 @@ def get_text(self):
 class LTComponent(LTItem):
     """Object with a bounding box"""
 
-    def __init__(self, bbox):
+    def __init__(self, bbox, orig_bbox=None):
```

Comment:
This naming is unclear. Can you please comment what bbox and orig_bbox are?

Reply:
Or even better, name them in a clear way

Reply:
Sooo Lifan asked me to add this code in (he sent me a diff). I don't actually know why it's here. My guess is that it uses pdfminer to extract both the original bounding box (provided by pdfminer code) and the new "tight" bounding box computed by our specialized logic glyph logic. And downstream logic in the code I can't find decides which to use.  I can't find a corresponding PR that uses it though. Do you happen to know?

Reply:
Ah found it https://github.com/vannevar-labs/arnold/pull/980/files